inc/saistatus.h

```diff
@@ -173,6 +173,11 @@
  */
 #define SAI_STATUS_NOT_EXECUTED                     SAI_STATUS_CODE(0x00000017L)
 
+/**
+ * @brief Invalid object location
+ */
+#define SAI_STATUS_STAGE_MISMATCH                   SAI_STATUS_CODE(0x00000018L)
```

Comment:
description don't match define name, maybe explain it more here why invalid object location is same as stage mismatch